commcare_export/__init__.py

```diff
@@ -13,9 +16,26 @@ def write(self, buf):
             self.logger.log(self.level, line.rstrip())
 
 
+def logger_name_from_filepath(filepath):
```

Comment:
A test for this would be cool. ... Also, I suspect unusual things happen to filenames that start with "py", like "commcare_export/pyexample.py".

Reply:
See [af99b49](https://github.com/dimagi/commcare-export/pull/233/commits/af99b494b5bea41d595612ca3f9a98c25b51d119)